ee/clickhouse/models/cohort.py

```diff
@@ -157,7 +159,7 @@ def get_entity_cohort_subquery(cohort: Cohort, cohort_group: Dict, group_idx: in
 
         params: Dict[str, Union[str, int]] = {"count": int(count), **entity_params, **date_params}
 
-        return f"{'NOT' if is_negation else ''} person_id IN ({extract_person})", params
+        return f"{'NOT' if is_negation else ''} {custom_match_field} person_id IN ({extract_person})", params
```

Comment:
What's happening here? Seems like it's going to read `person_id person_id`?

Reply:
maybe that's where my copy paste failed

Reply:
updated, good catch, thanks!